bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -4078,6 +4078,19 @@ spec:
                     - fatal
                     - panic
                     type: string
+                  logTypes:
+                    default: FLOWS
+                    description: logTypes defines the desired record types to generate.
+                      Possible values are "FLOWS" (default) to export flowLogs, "CONVERSATIONS"
+                      to generate newConnection, heartbeat, endConnection events,
```

Comment:
should it be `Conversation` here too?

Reply:
without doing the FLP renaming, we will have to keep these events

let's secure this first and see how long it will take to merge FLP renaming ?